tests/test_search.py

```diff
@@ -1,24 +1,26 @@
+import os
 from io import BytesIO
 
 from lxml import etree
 
 
 def test_empty_response(client) -> None:
-    response = client.get('/search')
-    xml = etree.parse(BytesIO(response.data))
-    root = xml.getroot()
 
-    assert root.getchildren() == []
+    path = os.path.join(os.path.dirname(os.path.dirname(__file__)),
+                        'schemas',
+                        'item-list.xsd')
+
+    with open(path, 'rb') as xmlfile:
+        xmlschema_doc = etree.XML(xmlfile.read())
 
+    xmlschema = etree.XMLSchema(xmlschema_doc)
 
-def test_empty_response_is_well_formed(client) -> None:
```

Comment:
I would still keep the two tests separate: schema validation and empty list are different concerns right?

Reply:
as in, schema validation is a "well-formed" check, but doesn't place constraints on the content.

When we have multiple tests, we could coinceivably extract schema validation and execute it _as part of every test_, if it's worth the effort

Reply:
The idea is that in every test the response is checked against the schema and then checks the specifics of the test because the schema contains information about the individual elements that save writing assertions for. For example, the `id` attribute of `item-ref` checks that it's an positive integer.

Reply:
>When we have multiple tests, we could coinceivably extract schema validation and execute it _as part of every test_, if it's worth the effort

Yes I think this would be a good idea as the tests grow. Consider this PR the first step towards better testing for search.

Reply:
👍 to doing them in combo.

Reply:
Ok, the next step would then be to extract the XML validation for reuse across tests